data/settings/1.32.x/aws.toml

```diff
@@ -2,6 +2,38 @@
 description = """
 The base64-encoded representation of data used to populate `~/.aws/config`
 """
+
+default = """
+A minimal configuration to enable or disable the [AWS FIPS endpoints], depending on the [variant flavor]
+
+[AWS FIPS endpoints]: https://docs.aws.amazon.com/general/latest/gr/rande.html#FIPS-endpoints
+[variant flavor]: ../../../concepts/variants/#flavors
+"""
+
+note = """
+Bottlerocket FIPS variants are available all AWS and AWS GovCloud regions. 
+However, in regions without FIPS support (those outside of the US and Canada), **you must opt-out** from using [AWS FIPS endpoints] by updating the `settings.aws.confg` API with the encoded AWS config to disable AWS FIPS endpoints.
+You can use the following values in your nodes to opt-out from using FIPS endpoints in AWS API calls.
+
+```toml
+[settings.aws]
+config = "W2RlZmF1bHRdCnVzZV9maXBzX2VuZHBvaW50PWZhbHNl"
+```
+
+```shell
+apiclient set settings.aws.config="W2RlZmF1bHRdCnVzZV9maXBzX2VuZHBvaW50PWZhbHNl"
+```
+
+The `base64` string corresponds to the following configuration:
+
+```text
```

Comment:
Missed this one in the previous PR. Can we replace `text` with `toml`?

Reply:
Let's update this one in all the versions.

Reply:
This isn't technically toml. I can do this in AWS configs:

```
[profile user1]
```

Which isn't valid toml